x-pack/plugins/security_solution/public/common/containers/matrix_histogram/index.ts

```diff
@@ -98,10 +103,12 @@ export const useMatrixHistogram = ({
             next: (response) => {
               if (isCompleteResponse(response)) {
                 if (!didCancel) {
-                  const histogramBuckets: Array<{
-                    key: string;
-                    doc_count: number;
-                  }> = getOr([], 'rawResponse.aggregations.eventActionGroup.buckets', response);
+                  const bucketEmpty: Buckets = [];
```

Comment:
@XavierM please let me know if this is what you were thinking.

Reply:
Yes but the `const bucketEmpty` need to go on line 34 ;)